alpakka/src/main/scala/org/scanamo/ops/retrypolicy/RetryPolicy.scala

```diff
@@ -40,6 +46,11 @@ sealed abstract class RetryPolicy extends Product with Serializable { self =>
 
   final def &&(that: RetryPolicy): RetryPolicy = And(self, that)
   final def ||(that: RetryPolicy): RetryPolicy = Or(self, that)
+
+  final private def getDelayWithJitter(delayInMillis: Long) = {
+    val maxJitter = Math.ceil(delayInMillis * 0.2).toInt
```

Comment:
I would instead force the user to supply lower and upper bounds instead of _[0..delay*0.2]_, for three reasons:
- some users may not want any jitter at all
- some users may want to introduce jitter no matter what, say minimum 10ms
- having an upper bound calculated from the delay feels rather arbitrary

Reply:
@regiskuckaertz So as you have suggested we have updated the jitter, So now we have lower and upper bound and graph will line between lower and upper bound for example: -

case1: -
```
delay = 1000ms
lowerBound = 0
upperBound = 0
delayWithJitter = 1000ms
```

case2:-
```
delay = 1000
lowerBound = 100
upperBound = 100
delayWithJitter = random value in range[900, 1100]
```

case3:-
```
delay = 1000
lowerBound = 0
upperBound = 100
delayWithJitter = random value in range[1000, 1100]
```

case4:-
```
delay = 1000
lowerBound = 100
upperBound = 100
delayWithJitter = random value in range[900, 1000]
```

Reply:
Great stuff! What is the advantage for having that level of precision over the "direction" of the jitter?